app/Checkers/Certificate.php

```diff
@@ -58,6 +44,58 @@ private function fetch()
             $ignoreMismatch = false
         );
 
+        # Do proper SSL Check of whole chain
```

Comment:
Can we "tidy" this whole block up to keep the code style more consistent with the rest of the project please?

e.g. single quotes, removing old array syntax, lowercase booleans etc?

Reply:
Should be conform with your code style now :)